web/app/scripts/navbar/module.js

```diff
@@ -2,6 +2,9 @@
     'use strict';
 
     angular.module('driver.navbar', [
+        'driver.resources.recordstate',
+        'driver.resources.polygonstate',
+        'driver.resources.geographystate',
```

Comment:
I like this idea too; there seems to be a fair amount of duplication across the three modules the way it's currently structured.